src/pages/Auth/Login.js

```diff
@@ -0,0 +1,176 @@
+import {
+  Stack,
+  Box,
+  Typography,
+  Divider,
+  TextField,
+  styled,
+  Button,
+  InputAdornment,
+  IconButton,
+  FormControl,
+  InputLabel,
+  OutlinedInput,
+} from "@mui/material";
+import react, { useState } from "react";
+import { Link } from "react-router-dom";
+import bgImg from "../../Assets/Images/bg-collage.jpg";
+import blitheLogo from "../../Assets/logo/logo-blue.jpg";
+import { useThemeMode } from "../../Helpers/Context";
+// import { SwipeableStepper } from '../Shared';
+import "./Auth.css";
+import Visibility from "@mui/icons-material/Visibility";
+import VisibilityOff from "@mui/icons-material/VisibilityOff";
+import { Input } from "../Shared";
+
+
+const InputBox = styled(Box)(({ theme }) => ({
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+  gap: "10px",
+}));
+
+const CustomStack = styled(Box)(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  justifyContent: "center",
+  alignItems: "center",
+  height: "100vh",
+  color:
+    theme.palette.mode === "light"
+      ? theme.palette.primary
+      : theme.palette.text.primary,
+  [theme.breakpoints.down("sm")]: {
+    position: "relative",
+    zIndex: 2,
+  },
+}));
+
+const AuthContainer = styled(Box)(({ theme }) => ({
+  backgroundColor:
+    theme.palette.mode === "light"
+      ? "#F5F5F5"
+      : theme.palette.background.default,
+  color: "#A14F57",
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "center",
+  alignItems: "center",
+  gap: "5px",
+  width: "35%",
+  height: "90%",
+  margin: "5px",
+  [theme.breakpoints.down("sm")]: {
+    position: "absolute",
+    zIndex: 2,
+    width: "80%",
+    backgroundColor:
+      theme.palette.mode === "light"
+        ? theme.palette.background.primary
+        : theme.palette.background.default,
+    opacity: 0.8,
+    height: "80%",
+  },
+}));
+
+const Login = () => {
+
+  const [values, setValues] = useState({
+    email: "",
+    password: "",
+    show_password: "false",
+    button_disable: false,
+    error: { username: null, password: null },
+  });
+
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    setValues({
+      ...values,
+      [name]: value,
+    });
+  };
+
+  const handleshowPassword = () => {
+    setValues((prevState) => {
+      return {
+        ...prevState,
+        show_password: prevState.show_password==='true' ? "false" : "true",
+      };
+    });
+  };
+
+  const handleLogin = (e) => {
+    e.preventDefault();
+    console.log("handle login..");
+    console.log("values is:", values);
+  };
+
+  console.log("values:",values);
```

Comment:
Suggestion: just a remainder to remove console.log when finished working

Reply:
ok